### PR TITLE
fix: add padding to InputField text suffix

### LINKED ISSUE #2308: Fix padding on InputField suffix
As seen at [orbit.kiwi](https://orbit.kiwi/components/inputfield/#suffixes), text suffixes for InputFields don't have the proper right padding and overlap the edge of the field.

### PR DESCRIPTION
**Pull request by simonhoyos**
Fix `InputField` suffix padding in the docs.

- [x] Updated InputField example to show how to add padding to text suffix. (as current storybook example)
- [x] Built examples.

Fixes #2308

### CODE REVIEW

File: packages/orbit-components/examples/BadgePrimitive/DEFAULT.json [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
-	"imports": "import BadgePrimitive from \"@kiwicom/orbit-components/lib/BadgePrimitive\";\nimport Stack from \"@kiwicom/orbit-components/lib/Stack\";\nimport Train from \"@kiwicom/orbit-components/lib/icons/Train\";\nimport Passengers from \"@kiwicom/orbit-components/lib/icons/Passengers\";",
-	"example": "() => (\n  <Stack direction=\"column\">\n    <BadgePrimitive>Tip</BadgePrimitive>\n    <BadgePrimitive ariaLabel=\"Train\" icon={<Icons.Train />} />\n    <BadgePrimitive ariaLabel=\"4 passengers\" icon={<Icons.Passengers />}>\n      4\n    </BadgePrimitive>\n  </Stack>\n)\n",
+	"imports": "import IllustrationPrimitive from \"@kiwicom/orbit-components/lib/IllustrationPrimitive\";",
```
Comment (vepor) [inline packages/orbit-components/examples/BadgePrimitive/DEFAULT.json:2]: This seems odd. Could you revert these changes? Otherwise this PR is ready to merge!

**Comment by vepor** [comment]
@CollierCZ for the general problem with `suffix` property, I've created internal task [ORBIT-1445](https://jira.kiwi.com/browse/ORBIT-1445).

File: packages/orbit-components/examples/BadgePrimitive/DEFAULT.json (hunk 1 above)
Comment (simonhoyos) [reply packages/orbit-components/examples/BadgePrimitive/DEFAULT.json:2]: The changes made to DEFAULT.json and SUFFIX.json were autogenerated  by the script `build:examples`. Do you want me to delete both files or just this one?

**Comment by CollierCZ** [comment]
Can we at least add a note to the props table in the readme? Or maybe in the Functional specs section, since there's already something about suffix there.

File: packages/orbit-components/examples/BadgePrimitive/DEFAULT.json (hunk 1 above)
Comment (CollierCZ) [reply packages/orbit-components/examples/BadgePrimitive/DEFAULT.json:2]: Just the DEFAULT.json file. That must be the result of a problem elsewhere and shouldn't be checked in.

Comment (simonhoyos) [reply packages/orbit-components/examples/BadgePrimitive/DEFAULT.json]: Done

**Review by vepor — APPROVED** [review]
